src/api/services/configs.ts

```diff
@@ -123,6 +123,17 @@ export const getConfigChanges = (
   );
 };
 
+export const getConfigList = (type?: string) => {
+  return resolve<Pick<ConfigItem, "id" | "name" | "config_class" | "type">[]>(
```

Comment:
I think we need icon here as well

Reply:
@moshloop  Don't we use the config_class and type for icons for configs?